src/components/index/Index.styled.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from "@emotion/styled";
+
+export const Wrapper = styled.div`
+  background-color: #efefef;
+  min-height: 100vh;
+  padding-bottom: 1rem;
+`;
+
+export const Header = styled.div`
+  background-color: #272528;
+`;
+
+export const HeaderTitle = styled.div`
+  max-width: 1400px;
+  margin: 0 auto;
+  padding: 2rem 0;
+`;
+
+export const HeaderTitleText = styled.span`
+  font-size: 2.25rem;
+  font-family: "Halyard";
+  font-weight: 600;
+  color: #fff;
+`;
+
+export const HeaderTitleTextRed = styled(HeaderTitleText)`
+  color: #ff4a4a;
```

Comment:
nit: hardcoded colors

Reply:
I haven't defined the css variables yet -- might just be better to do this in one go in a separate ticket.